rllib/utils/actor_manager.py

```diff
@@ -166,30 +182,63 @@ def __init__(
                 that cannot be scheduled because the limit has been reached will be
                 dropped. This only applies to the asynchronous remote call mode.
         """
-        self.__actors = actors or []
-        # Start with healthy state for all remote actors.
-        self.__remote_actor_states = [True] * len(self.__actors)
-
-        # Collection of outstanding async requests, mapping from actor index
-        # to a list of pending requests.
-        self.__remote_reqs_in_flight: Mapping[int, int] = defaultdict(lambda: 0)
-        # Maps outstanding async requests to the indices of the actors that
+        self.__NEXT_ID = 0
+
+        # Actors are stored in a map and indexed by a unique id.
+        self.__actors = {}
```

Comment:
Can clarify mapping is from int to Actors?

Reply:
`Mapping[int, Actor]`

Reply:
good catch.